{"clientVersion":"6.15.0","errorCode":"P1001","level":"error","message":"Database connection failed: Can't reach database server at `localhost:5432`\n\nPlease make sure your database server is running at `localhost:5432`.","name":"PrismaClientInitializationError","service":"ai-chain-creator-hub-backend","stack":"PrismaClientInitializationError: Can't reach database server at `localhost:5432`\n\nPlease make sure your database server is running at `localhost:5432`.\n    at r (/workspaces/AI-Chain-Creator-Hub/packages/backend/node_modules/@prisma/client/src/runtime/core/engines/library/LibraryEngine.ts:440:17)\n    at connectDatabase (/workspaces/AI-Chain-Creator-Hub/packages/backend/src/config/database.ts:32:5)\n    at startServer (/workspaces/AI-Chain-Creator-Hub/packages/backend/src/index.ts:60:5)","timestamp":"2025-09-04T23:16:18.796Z"}
{"level":"error","message":"Error sending notification: Push SDK Error: API : channel::send : AxiosError: Request failed with status code 400","service":"ai-chain-creator-hub-backend","stack":"Error: Push SDK Error: API : channel::send : AxiosError: Request failed with status code 400\n    at Channel.<anonymous> (/workspaces/AI-Chain-Creator-Hub/packages/backend/packages/restapi/src/lib/pushNotification/channel.ts:169:13)\n    at Generator.throw (<anonymous>)\n    at rejected (/workspaces/AI-Chain-Creator-Hub/packages/backend/node_modules/tslib/tslib.js:168:69)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)","timestamp":"2025-09-06T04:32:39.317Z"}
{"level":"error","message":"Error sending notification: Push SDK Error: API : channel::send : AxiosError: Request failed with status code 400","service":"ai-chain-creator-hub-backend","stack":"Error: Push SDK Error: API : channel::send : AxiosError: Request failed with status code 400\n    at Channel.<anonymous> (/workspaces/AI-Chain-Creator-Hub/packages/backend/packages/restapi/src/lib/pushNotification/channel.ts:169:13)\n    at Generator.throw (<anonymous>)\n    at rejected (/workspaces/AI-Chain-Creator-Hub/packages/backend/node_modules/tslib/tslib.js:168:69)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)","timestamp":"2025-09-06T04:32:39.323Z"}
{"level":"error","message":"Error creating channel: Push SDK Error: Contract : createChannelWithPUSH : Error: {\"code\":\"UNSUPPORTED_OPERATION\",\"operation\":\"call\",\"shortMessage\":\"missing provider\"}","service":"ai-chain-creator-hub-backend","stack":"Error: Push SDK Error: Contract : createChannelWithPUSH : Error: {\"code\":\"UNSUPPORTED_OPERATION\",\"operation\":\"call\",\"shortMessage\":\"missing provider\"}\n    at Channel.<anonymous> (/workspaces/AI-Chain-Creator-Hub/packages/backend/packages/restapi/src/lib/pushNotification/channel.ts:271:13)\n    at Generator.throw (<anonymous>)\n    at rejected (/workspaces/AI-Chain-Creator-Hub/packages/backend/node_modules/tslib/tslib.js:168:69)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)","timestamp":"2025-09-06T04:32:40.260Z"}
{"level":"error","message":"Error creating channel: Push SDK Error: Contract : createChannelWithPUSH : Error: {\"code\":\"UNSUPPORTED_OPERATION\",\"operation\":\"call\",\"shortMessage\":\"missing provider\"}","service":"ai-chain-creator-hub-backend","stack":"Error: Push SDK Error: Contract : createChannelWithPUSH : Error: {\"code\":\"UNSUPPORTED_OPERATION\",\"operation\":\"call\",\"shortMessage\":\"missing provider\"}\n    at Channel.<anonymous> (/workspaces/AI-Chain-Creator-Hub/packages/backend/packages/restapi/src/lib/pushNotification/channel.ts:271:13)\n    at Generator.throw (<anonymous>)\n    at rejected (/workspaces/AI-Chain-Creator-Hub/packages/backend/node_modules/tslib/tslib.js:168:69)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)","timestamp":"2025-09-06T04:32:40.261Z"}
